packages/oui-datagrid/README.md

```diff
@@ -127,6 +127,38 @@ Clicked row action 1: <span ng-if="$ctrl.action1Row">{{$ctrl.action1Row.lastName
   When <code class="oui-doc-codespan">selectable-rows</code> is provided, you can listen to checkboxes changes with the <code class="oui-doc-codespan">on-row-select</code> event. It provides <strong>$row</strong> as the triggered row and <strong>$rows</strong> as the complete list of rows selected so far.
 </oui-message>
 
+#### Keep selection across pages 
+
+```html:preview
+<oui-datagrid rows="$ctrl.data" page-size="5" selectable-rows on-row-select="$ctrl.onRowSelect($row, $rows)" keep-selected-rows>
```

Comment:
Shouldn't `keep-selected-rows` be the default option for version 3 ?

Also, `selectable-rows`, `on-row-select` and `keep-selected-rows` don't look alike at all but they all concern the same feature, maybe you could standardize the naming ? `selectable-rows-retaining` ?